src/Settings/KineticGrid.cpp

```diff
@@ -212,13 +214,26 @@ FVM::PXiGrid::PXiMomentumGrid *SimulationGenerator::Construct_PXiGrid(
             xgg = new FVM::PXiGrid::XiUniformGridGenerator(nxi);
             break;
 
-        case OptionConstants::PXIGRID_XITYPE_BIUNIFORM: {
+        case OptionConstants::PXIGRID_XITYPE_BIUNIFORM:{ 
             real_t xisep = s->GetReal(mod + "/xisep");
             len_t nxiSep = s->GetInteger(mod + "/nxisep");;
 
             xgg = new FVM::PXiGrid::XiBiUniformGridGenerator(nxi, nxiSep, xisep);
-            break;
-        }
+		} break;
```

Comment:
The curly bracket should be aligned with the start of the ``case`` keyword on line 217.